src/Keboola/DbExtractor/Extractor/Snowflake.php

```diff
@@ -415,9 +446,15 @@ public function getTables(array $tables = null)
             "SELECT * FROM information_schema.columns 
              WHERE TABLE_NAME IN (%s)
              ORDER BY TABLE_SCHEMA, TABLE_NAME, ORDINAL_POSITION",
-            implode(', ', array_map(function ($tableName) {
-                return "'" . $tableName . "'";
-            }, $tableNameArray))
+            implode(
+                ', ',
+                array_map(
+                    function ($tableName) {
+                        return "'" . $tableName . "'";
```

Comment:
why not quoteIdentifier?

Reply:
it's not an identifier in this case, but a value

Reply:
I see, but anyway... isn't there a proper quoting function? This is not quoting, this is "wrapping in quotes".

Reply:
I don't think there is a "proper" one, but there is a private method in this class that should be used here, https://github.com/keboola/db-extractor-snowflake/blob/master/src/Keboola/DbExtractor/Extractor/Snowflake.php#L538

Reply:
If there is nothing better... 

> The addslashes() is sometimes incorrectly used to try to prevent SQL Injection. Instead, database-specific escaping functions and/or prepared statements should be used.
> -- http://php.net/manual/en/function.addslashes.php

Reply:
yes, that's a completely valid point, I'll change it to statement.  thanks.

Reply:
I'll change it when I fix this: https://github.com/keboola/db-extractor-snowflake/issues/72